spec/ttml1.xml

```diff
@@ -13244,6 +13243,53 @@ shown in <specref ref="fragment-streaming-graphic"/>.</p>
 <p>This specification does not define a transport buffer model or a decoder
 capabilities model.</p>
 </note>
+</div2>
+<div2 id="temporal-fragmentation">
+<head>Temporal fragmentation</head>
+<p>Another possible means by which TTML Content may be streamed is to
+partition a <emph>Document Instance</emph>'s information set into
```

Comment:
Change ``<emph>Document Instance</emph>`` to a ``<loc ...>document instance</loc>`` link.